### ISSUE TITLE
G183: 4.5:1 in headline of example 1, 5:1 in the example

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#2897]
[headline example 1](https://www.w3.org/WAI/WCAG21/Techniques/general/G183): 

> Colors that would provide 3:1 contrast with black words and 4.5:1 contrast with a white background

[linked example 1](https://www.w3.org/WAI/WCAG21/working-examples/link-contrast/):

> The following 26 web-safe colors pass at 3:1 vs black and 5:1 vs. white.

What explains the difference between 4.5:1 and 5:1? 4.5:1 would be the requirements from SC 1.4.3. Is 5:1 a typo or deliberately chosen to ensure better visibility of the links? 
* If typo: Then had to check on fix if more colors are added if 4.5:1 instead of 5:1 is the limit.
* If on purpose: Then this should be explained. And besides, then e.g. 6:1 would be much better to distinguish the links.

Actually, it would be good to point out in the example that this only satisfies 1.4.3 and not 1.4.6. It can also be pointed out that AAA conformance cannot be achieved with G183, since there is no web-safe color that has 7:1 to the white background and 3:1 to the black text. 

By the way: Do the web-safe colors still have any relevance today? Or do not all devices display all colors correctly, regardless of whether web-safe or not?

**Comment by mbgower** [comment]
> Is 5:1 a typo or deliberately chosen to ensure better visibility of the links?

I have checked the colours supplied and they all exceed 5:1; given there are many colours that fall in between 4.5:1 and 5:1, and none of them are listed here, I am assuming that the use of "5:1" is deliberate.

> If on purpose: Then this should be explained. And besides, then e.g. 6:1 would be much better to distinguish the links.

I am not sure who created this list. I cannot really conjecture on the rationale. If you would prefer this to say "4.5:1" that would still be accurate.

> Actually, it would be good to point out in the example that this only satisfies 1.4.3 and not 1.4.6. 

This technique is sufficient for Use of Color. It is not listed as sufficient for either of the contrast requirements.

> By the way: Do the web-safe colors still have any relevance today? Or do not all devices display all colors correctly, regardless of whether web-safe or not?

As I understand it, web-safe colors are not relevant with current technologies. Others may wish to comment on any need to maintain for people still using technologies that were current at the time of this working example publication.

---

Please feel free to create a PR which you feel addresses your concerns.

**Comment by patrickhlauke** [comment]
This was addressed as part of https://github.com/w3c/wcag/pull/5186
